crates/wasm/src/verifier/mod.rs

```diff
@@ -49,8 +47,10 @@ impl State {
 impl JsVerifier {
     #[wasm_bindgen(constructor)]
     pub fn new(config: VerifierConfig) -> JsVerifier {
+        let tlsn_config = tlsn::verifier::VerifierConfig::builder().build().unwrap();
         JsVerifier {
-            state: State::Initialized(Verifier::new(config.into())),
+            state: State::Initialized(Verifier::new(tlsn_config)),
+            config,
```

Comment:
this was confusing to me. why not expose the config like we do in tlsn?

Reply:
It's because we're going to delete `tlsn-wasm`. I only made these changes to ensure it continued to compile and behave the same.